server/src/main/java/com/cloud/storage/VolumeApiServiceImpl.java

```diff
@@ -2235,6 +2236,12 @@ private void setNewIopsLimits(VolumeVO volume, DiskOfferingVO newDiskOffering, L
         }
     }
 
+    private void validateChangeDiskOffering(DiskOfferingVO existingDiskOffering, DiskOfferingVO newDiskOffering) {
```

Comment:
define this method in _VolumeApiService_ and use the same in _UserVMManagerImpl_.

Reply:
Please review the new changes @sureshanaparti